rootfs/api/models/appsettings.py

```diff
@@ -19,7 +19,9 @@ class AppSettings(UuidAuditedModel):
     app = models.ForeignKey('App', on_delete=models.CASCADE)
     maintenance = models.NullBooleanField(default=None)
     routable = models.NullBooleanField(default=None)
-    whitelist = ArrayField(models.CharField(max_length=50), default=[])
+    # the default values is None to differentiate from user sending an empty whitelist
+    # and user just updating other fields meaning the values needs to be copied from prev release
+    whitelist = ArrayField(models.CharField(max_length=50), default=None)
```

Comment:
this is just the default value, the value saved in database will be empty array....i am doing it update_whitelist function which is called before saving to database.